### ISSUE TITLE
List of devices not refreshed after umount if the list has a filter applied (search ny keyword)

### ISSUE THREAD

**Issue by moravekl** [FRINXio/frinx-uniconfig-ui#28]
Normally if I select devices to be unmounted and press "Unmount Devices" the list is refreshed to reflect that those devices are no longer in the list of mounted devices.

If I search for example for "Leaf" to find all leaf devices, select them and then press "Unmount Devices", nothing happens to the list, as if nothing was executed. To verify that the devices were unmounted, first I need to clear the filter to see that the leaf devices are no longer in the list.

Additionally, sometimes an error is thrown. (Not sure what are the exact conditions for this error to be thrown, it isn't thrown every time)

```
Unhandled Rejection (TypeError): Cannot read property '0' of undefined
(anonymous function)
src/components/uniconfig/deviceTable/List.js:62
  59 | //append os/version from conf
  60 | if (topology === "cli") {
  61 |     os_version = await http.get('/api/odl/conf/status/' + topology + "/" + node_id).then(res => {
> 62 |         os_version = res["node"]["0"]["cli-topology:device-type"];
     | ^  63 |         os_version = os_version + " / " + res["node"]["0"]["cli-topology:device-version"];
  64 |         return os_version;
  65 |     })
```

**Comment by ChmaraX** [comment]
Attached error fixed here: #33

**Comment by ChmaraX** [comment]
Unmount device error fixed here: https://github.com/FRINXio/frinx-uniconfig-ui/commit/9a263284adfd177c12c18092aafe75435d77d915
